cmasher/utils.py

```diff
@@ -233,6 +239,97 @@ def _get_cmap_perceptual_rank(
 
 
 # %% FUNCTIONS
+# This function combines multiple colormaps at given nodes
+def combine_cmaps(
+    cmaps: list[Union[Colormap, str]],
+    nodes: list[float] = None,
```

Comment:
Can you also give me some advice in addressing this `mypy` error for this line? `utils.py:245: error: Incompatible default for argument "nodes" (default has type "None", argument has type "list[float]")  [assignment]`

I cannot think of an elegant way to resolve this except for remove the default value `None` and force user input of `nodes`. But having a default `nodes` is preferred personally.

Reply:
this is the fix.
```suggestion
    nodes: Optional[list[float]] = None,
```

Reply:
Thanks @neutrinoceros! New trick learned.